src/rgw/rgw_op.h

```diff
@@ -1768,14 +1768,10 @@ class RGWSetAttrs : public RGWOp {
 
 class RGWGetObjLayout : public RGWOp {
 protected:
-  RGWRados::Object *target{nullptr};
   RGWObjManifest *manifest{nullptr};
   rgw_raw_obj head_obj;
 
 public:
-  RGWGetObjLayout() {
```

Comment:
Yes, Your fix doesn't need to remove the already existing constructor. It can be used to initialize variables later.